### ISSUE TITLE
Inline markup changed from url() to url(url))

### ISSUE THREAD

**Issue by will-ashworth** [Mecanik/cloudflare-image-resizing-worker#8]
Because of the double nesting going on, most images with Elementor are not loading. Some are (the ones using srcset attributes are fine), but we had to disable the worker because the current scripted solution is breaking the site.

Any idea on how we could tweak this to work more generically for most WordPress sites? I'll admit I'm new to workers with Cloudflare, and your solution looks robust, but we just couldn't continue with it in its current state 🫠

**Comment by pa2codes** [comment]
@will-ashworth try to remove url('') from regex in DivTagRewriter function. That is working for me :)

if (element.hasAttribute("style")) {
			const style = element.getAttribute("style");

			if (style) {
				if (style.indexOf(`/wp-content/`) !== -1 && style.indexOf('/cdn-cgi/image/') === -1) {
					const CDN = `/cdn-cgi/image/quality=${IMAGE_QUALITY},format=auto,onerror=redirect,metadata=none`;

					let result = style.replace(rgxSrc, `$1${CDN}$2$3`); //<- here I removed the url('')

					element.setAttribute("style", result);
				}
			}
		}

**Comment by Mecanik** [comment]
Please note:

```
element {
    background-image: url('path/to/image.jpg');
}
```

This is the more commonly used format and is typically recommended because it works universally, even if the URL contains special characters or spaces. I forcefully add quotes to fix any issues because many WP themes are specifying without quotes.

```
element {
    background-image: url(path/to/image.jpg);
}
```

This works for URLs that don't contain special characters or spaces. If the URL has any characters that might be misinterpreted in CSS, it's safer to use quotes.

The CSS specifications allow both formats, but for consistency and to avoid potential pitfalls, many developers opt to always use quotes. So am I.

In conclusion, the official way allows for both, but using quotes `url('path/to/image.jpg')` is generally recommended, so we are going with that.

I am uploading a new improvement for the CSS rewriting shortly.
